src/executables/unload_schema.c

```diff
@@ -3934,3 +4045,691 @@ create_filename (const char *output_dirname, const char *output_prefix, const ch
 
   return 0;
 }
+
+static int
+create_filename (const char *output_dirname, const char *output_prefix, const char *middle, const char *suffix,
+		 char *output_filename_p, const size_t filename_size)
+{
+  if (output_dirname == NULL)
+    {
+      output_dirname = ".";
+    }
+
+  size_t total = strlen (output_dirname) + strlen (output_prefix) + strlen (middle) + strlen (suffix) + 8;
+
+  if (total > filename_size)
+    {
+      return -1;
+    }
+
+  snprintf (output_filename_p, filename_size - 1, "%s/%s%s%s", output_dirname, output_prefix, middle, suffix);
+
+  return 0;
+}
+
+static int
+extract_only_user (extract_context & ctxt)
```

Comment:
You used function name like 'extract_only_user', '
extract_only_serial,
extract_only_class,
extract_only_synonym,
extract_only_procedure,
extract_only_server,
extract_only_vclass,
extract_only_pk,
extract_only_fk,
extract_only_grant

I'll take one example with **extract_only_grant**, following is preferred priority for me.
1. if it is applicable '**emit_grant**' use it
2. if 1 is not applicable then,  applicable '**extract_grant**'
3. nothing applicable, '**extract_grant_only**'

Reply:
Thanks for the advice.
I will refer to the above.